airqo-platform/AirQo-frontend
# Ft device monitoring

- What does this PR do?
This PR resolves issues that were identified on the forget password feature. It also creates all the UI elements for the Network Manager App.
![image](https://user-images.githubusercontent.com/1590213/79537487-01f71180-808b-11ea-9f03-9ba85f7a256b.png)


- What JIRA task is related to this PR?

[Unsent emails on Forget Password Feature](https://airqoteam.atlassian.net/browse/LOC-25?atlOrigin=eyJpIjoiODNlN2Y3N2ZiOGJhNDA1MzlmMTUyZjJjYmYyZDM2OGIiLCJwIjoiaiJ9)

- How do I test out this PR?

1. Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local` to test out the server that runs on port 5000. For windows users, you might want to first switch to a new testing branch and then editing the **package.json** script line from  `"local": "PORT=5000 react-scripts start"`, to  `"local": "react-scripts start"`,

2. Please ensure that you also have the auth service API running in another browser since it is consuming the localhost endpoints. That is: open a new tab of your terminal and: `cd AirQo-api/src/auth-servic`e and then run `npm install`. Finally, run` npm run dev` to get the API server running on port 3000. You will also need to have a .env file in this service for things to run well. Contents of this .env file can be requested for from Martin separately. 

- Which end points should are ready for testing?
There are no endpoints for this one.
